_gsocblogs/2022/blog_ROOT_Neelshah.md

```diff
@@ -0,0 +1,366 @@
+I have prepared a detailed blog on how did i got into GSOC 2022 at CERN HSF in detail [here](https://gist.github.com/Neel-Shah-29/3f04f05a8a353605068e32e55a5093c1)
+
+
+## Community Bonding Period
+
+<div align="center">
+<img src="https://miro.medium.com/max/1200/1*YvvzCmO4dbCPeTPOq7_OKA.png" alt="drawing" width="500"/>
+</div>
+
+This is the period of time between when accepted GSoC contributors are announced and the time they are expected to start coding. This time is an excellent one to introduce your GSoC contributors to the community, get them on the right mailing lists, introduce them to the codebase, discuss how they will work with their mentors on their timeline for the program, etc.
+
+Again I have written a very detailed blog on Community Bonding Period [here](https://gist.github.com/Neel-Shah-29/b2c887adaa118ba68b42e324d3b2a47a).
+
+### Brief Description of Work Done:-
+
+- For TMVA projects we usually have weekly meetings to discuss about the projects and solve the difficulties of everyone. So attending those meetings are mandatory as it involves your communication with other gsoc students of TMVA. Also the round table discussion solves all queries which you have as all the students and mentors give solutions and guidelines on it. Additionally we can also set a meeting with the mentor if required.
+
+- In TMVA projects, we have a specific idea of presenting our projects with the detailed timeline of implementation in front of mentors and other Gsoc students so we can get familier with other's project as well and by that time our concrete strategy of project is also ready.
+
+  Here is my presentation for reference:- [Inference Code Generation for Deep Learning models](https://docs.google.com/presentation/d/1bl1Xk5esr5TJEHKoro7o05pRQ2J997Qb1hc68XfU_0s/edit?usp=sharing)
+
+- I built and set up the Root environment in my laptop as required. [Here](https://root.cern/install/build_from_source/) is the link to build Root from source.
+
+- Finalised the evaluation dates so that i get enough time to complete my long term project. I chose a 16 weeks project after discussing with the mentor as my college reopens from July 18 and i will get less time to devote after that. 
+
+- Lastly, I began coding part for one of the easy operator which is Leaky Relu so i get an headstart for the Project. Here is the link to the PR :- https://github.com/root-project/root/pull/10415
+
+**Pro-Tip** :- Use this Period very well to connect with your mentors, other Gsoc Candidates and the organisation as a whole!
+
+## Coding Period 
+
+The first 3 PRs were implemented in the Community Bonding Period. 
+
+#### 1) Fixed a bug related to build in RModelParser_ONNX.cxx ( when building in Ubuntu 20.04)
+![image](https://user-images.githubusercontent.com/84740927/159856024-8b399e36-d835-4726-b6dc-9d1a5b90dbc3.png)
```

Comment:
Having the stack traces of bugs in the blog may be too much for a reader who wants a quick view of what you did. I would leave this kind of level of detail for a report.